dclab/features/emodulus/viscosity.py

```diff
@@ -142,7 +164,28 @@ def get_viscosity_mc_pbs_buyukurganci_2022(
         temperature: float = 23.0):
     """Compute viscosity of MC-PBS according to :cite:`Buyukurganci2022`"""
     check_temperature("'buyukurganci-2022' MC-PBS", temperature, 22, 37)
-    raise NotImplementedError("Model buyukurganci-2022 not implemented yet!")
+    # material constants for temperature behavior of MC dissolved in PBS:
+    alpha = 2.23
```

Comment:
According to the arxiv paper https://arxiv.org/abs/2302.01664 `alpha` should be 0.00223, correct?